src/awx_plugins/credentials/github_app.py

```diff
@@ -132,18 +132,19 @@ class EmptyKwargs(TypedDict):
 
 GH_CLIENT_ID_TRAILER_LENGTH = 16
 HEXADECIMAL_BASE = 16
+GH_CLIENT_ID_LENGTH_BOUNDS = (19, 22)
```

Comment:
Let's unbreak this by applying the original suggestion I made in https://github.com/ansible/awx-plugins/pull/139#discussion_r2473386861
```suggestion
GH_CLIENT_ID_LENGTH_BOUNDS = range(19, 23)  # min 19, max 22
```

`thing in (1, 42)` will only be true if `thing == 1 or thing == 42`. Using the range allows anything in-between. Just take into account that the second argument is not included in the range. So `3 in range(1, 42) is True` and `41 in range(1, 42) is True`, but `42 in range(1, 42) is False`.